client/src/api.ts

```diff
@@ -101,22 +101,22 @@ export type ENSQueryResult = { valid: false } | { valid: true; address: string }
 
 export type AddressQueryResult = { valid: false } | { valid: true; ens: string };
 
-let API_BASE = 'https://api.poap.xyz';
+// let API_BASE = 'https://api.poap.xyz';
 
-if (process.env.NODE_ENV === 'development') {
-  if (process.env.REACT_APP_API_ROOT) {
-    API_BASE = process.env.REACT_APP_API_ROOT;
-  } else {
-    API_BASE = 'http://localhost:8080';
-  }
-}
+let API_BASE = 'http://10.0.0.146:8080';
+// if (process.env.NODE_ENV === 'development') {
+//   if (process.env.REACT_APP_API_ROOT) {
+//     API_BASE = process.env.REACT_APP_API_ROOT;
+//   } else {
+//     API_BASE = 'http://localhost:8080';
+//   }
+// }
```

Comment:
This code will remain commented or can we safely remove it ?

Reply:
this is because the poap endpoint is not yet up with the latest changes Nahue has on his branch. This is production code and should be kept there. Once Nahue's branch is merged to production, we should uncomment this code and everything will work as intended. Do you think there is a better way to archive this?